### PR TITLE
GDB-8252 Fix displaying of additional Ontop properties

### PR DESCRIPTION
**Pull request by yordanalexandrov**
## What
Upon editing a repository, the additional Onto properties are not displayed correct. Instead, [object Object] is shown in the text area.

## Why
The server returns an object which is not parsed and is set in the textbox. The initial talk was for the UI to receive a string so there is no need to parse it, but this has changed.

## How
- added function to convert additional properties values to string

### CODE REVIEW

File: test-cypress/integration/repository/repositories.spec.js [hunk 1]
```diff
@@ -386,7 +386,7 @@ describe('Repositories', () => {
     });
 
     //Create Ontop repository and test ontop functionality
-    it('should create an Ontop repository', () => {
+    it.only('should create an Ontop repository', () => {
```
Comment (teodossidossev) [inline test-cypress/integration/repository/repositories.spec.js:389]: only

File: src/js/angular/repositories/controllers.js [hunk 2]
```diff
@@ -43,6 +43,14 @@ const parseMemberParamIfNeeded = function (param) {
     }
 };
 
+const convertAdditionalPropertiesParamIfNeeded = function (params) {
+    if (params.additionalProperties && params.additionalProperties.value) {
+        const values = params.additionalProperties.value;
```
Comment (teodossidossev) [inline src/js/angular/repositories/controllers.js:48]: are you sure what object is `values` ?

Comment (yordanalexandrov) [reply src/js/angular/repositories/controllers.js:50]: Added a check if values is an Object

**Comment by teodossidossev** [comment]
lgtm

**Review by teodossidossev — APPROVED** [review]

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**